data/pilots/galactic-republic/clone-z-95-headhunter.json

```
@@ -0,0 +1,180 @@
+{
+  "name": "Clone Z-95 Headhunter",
+  "xws": "clonez95headhunter",
+  "size": "Small",
+  "dial": [
+    "1BW",
+    "1FB",
+    "1NW",
+    "2TW",
+    "2BB",
+    "2FB",
+    "2NB",
+    "2YW",
+    "3TW",
+    "3BW",
+    "3FB",
+    "3NW",
+    "3YW",
+    "3KR",
+    "4FW",
+    "4KR"
+  ],
+  "dialCodes": ["Z95"],
+  "faction": "Galactic Republic",
+  "stats": [
+    { "arc": "Front Arc", "type": "attack", "value": 2 },
+    { "type": "agility", "value": 2 },
+    { "type": "hull", "value": 2 },
+    { "type": "shields", "value": 2 }
+  ],
+  "actions": [
+    { "difficulty": "White", "type": "Focus" },
+    { "difficulty": "White", "type": "Lock" },
+    { "difficulty": "Red", "type": "Barrel Roll" }
+  ],
+  "icon": "???",
+  "pilots": [
+    {
+      "name": "\"Killer\"",
+      "caption": "Dependable Closer",
+      "initiative": 2,
+      "limited": 1,
+      "cost": 0,
+      "loadout": 0,
+      "xws": "killer",
+      "ability": "While you perform an attack, if the defender has 2 or fewer hull remaining, you may roll 1 additional attack die. If you do, gain 1 deplete token after rolling attack dice.",
+      "shipAbility": {
+        "name": "Versatile Frame",
+        "text": "You can eqiup 1 [Torpedo] or [Missile] upgrade. While your revealed maneuver is blue, add white [Boost] to your action bar."
```

I attempted to just change this everywhere and push to this branch, GitHub didn't love that idea 😆 

```suggestion
        "text": "You can equip 1 [Torpedo] or [Missile] upgrade. While your revealed maneuver is blue, add white [Boost] to your action bar."
```